packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -40,6 +41,7 @@
                 v-else-if="isSelectionControl"
                 :id="uniqueId"
                 :attributes="$attrs"
+                :is-visually-required="isVisuallyRequired"
```

Comment:
As the props are already being bound to this component (on line 45), shouldn't `isVisuallyRequired` already be available in the `form-selection-control` component?